Vagrantfile

```diff
@@ -0,0 +1,36 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.network "forwarded_port", guest: 5000, host: 5000, auto_correct: true
+  
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+  sudo apt-get update
+  sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+  libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+  xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+  if [ -d "$HOME/.pyenv" ] ;then
+    echo "pyenv is already installed" 
+  else
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+    echo 'PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.profile
+    source ~/.profile
+    pyenv install 3.8.5
+    pyenv global 3.8.5
+  fi
```

Comment:
Lines 6-21 should be indented another level. 

These lines are part of calling the `config.vm.provision` function on line 5 - it's the value of the `inline` parameter.

Similarly the value of `inline` in trigger.run_remote should be indented.